app/ore/permission/role/RoleTypes.scala

```diff
@@ -22,6 +22,7 @@ object RoleTypes extends Enumeration {
   val CommunityLeader = new  RoleType( 4, 59, classOf[GlobalRole], Default,  "Community Leader",   Amber)
   val Staff           = new  RoleType( 5, 3,  classOf[GlobalRole], Default,  "Sponge Staff",       Amber)
   val SpongeDev       = new  RoleType( 6, 41, classOf[GlobalRole], Default,  "Sponge Developer",   Green)
+  val OreDev          = new  RoleType(26, 66, classOf[GlobalRole], Default,  "Ore Developer",      Orange)
```

Comment:
The ID for OrganisationAdmin is 26 too. This ends up in an error.

Reply:
:man_facepalming: